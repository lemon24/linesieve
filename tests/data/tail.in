--section '(\S+):$'
tail -s aaa -n1
tail -s bbb -n+2
tail -s ccc -n-1
tail -s ddd -n0
---
aaa:
1a
2a
3a
4a
5a
bbb:
1b
2b
3b
4b
5b
ccc:
1c
2c
3c
4c
5c
ddd:
1d
2d
