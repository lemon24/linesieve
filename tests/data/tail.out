aaa
5a
bbb
2b
3b
4b
5b
ccc
5c
ddd
